.github/workflows/admin.yml

```diff
@@ -43,15 +43,15 @@ jobs:
       - name: Add secrets to env file
         working-directory: admin
         run: |
-          echo REACT_APP_ADMIN_FB_API_KEY=${{ secrets.REACT_APP_ADMIN_FB_API_KEY }} > .env
-          echo REACT_APP_ADMIN_FB_AUTH_DOMAIN=${{ secrets.REACT_APP_ADMIN_FB_AUTH_DOMAIN }} >> .env
-          echo REACT_APP_ADMIN_FB_DATABASE_URL=${{ secrets.REACT_APP_ADMIN_FB_DATABASE_URL }} >> .env
-          echo REACT_APP_ADMIN_FB_MEASUREMENT_ID=${{ secrets.REACT_APP_ADMIN_FB_MEASUREMENT_ID }} >> .env
-          echo REACT_APP_ADMIN_FB_MESSAGING_SENDER_ID=${{ secrets.REACT_APP_ADMIN_FB_MESSAGING_SENDER_ID }} >> .env
-          echo REACT_APP_ADMIN_FB_PROJECT_ID=${{ secrets.REACT_APP_ADMIN_FB_PROJECT_ID }} >> .env
-          echo REACT_APP_ADMIN_FB_STORAGE_BUCKET=${{ secrets.REACT_APP_ADMIN_FB_STORAGE_BUCKET }} >> .env
-          echo REACT_APP_ADMIN_ALGOLIA_APPLICATION_ID=${{ secrets.REACT_APP_ADMIN_ALGOLIA_APPLICATION_ID }} >> .env
-          echo REACT_APP_ADMIN_ALGOLIA_SEARCH_KEY=${{ secrets.REACT_APP_ADMIN_ALGOLIA_SEARCH_KEY }} >> .env
+          echo VITE_ADMIN_FB_API_KEY=${{ secrets.REACT_APP_ADMIN_FB_API_KEY }} > .env
```

Comment:
Are the GitHub secrets now a bit inconsistent with the local / prod envs? Not a tragedy, though.

Reply:
Yeah. It's a little mess now, but still completely transparent and totally fine for me...